devicetypes/smartthings/zigbee-siren-src/zigbee-siren.groovy

```diff
@@ -0,0 +1,150 @@
+import physicalgraph.zigbee.zcl.DataType
+
+metadata {
+	definition(name: "ZigBee Siren", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "x.com.st.d.siren") {
+		capability "Actuator"
+		capability "Alarm"
+		capability "Configuration"
+		capability "Health Check"
+		capability "Refresh"
+
+		fingerprint profileId: "0104", inClusters: "0000,0003,0500,0502", outClusters: "0000", manufacturer: "ClimaxTechnology", model: "SRAC_00.00.00.16TC", deviceJoinName: "Ozom Smart Siren" // Ozom Siren - SRAC-23ZBS
+	}
+
+	tiles {
+		standardTile("alarm", "device.alarm", width: 2, height: 2) {
+			state "off", label:'off', action:'alarm.siren', icon:"st.secondary.siren", backgroundColor:"#ffffff"
+			state "siren", label:'siren!', action:'alarm.off', icon:"st.secondary.siren", backgroundColor:"#e86d13"
+		}
+
+		standardTile("refresh", "device.refresh", inactiveLabel: false, decoration: "flat") {
+			state "default", label: '', action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+
+		main "alarm"
+		details(["alarm","refresh"])
+	}
+}
+
+def installed() {
+	response(refresh())
+}
+
+def parse(String description) {
+	log.debug "parse ${description}"
+
+	def map = zigbee.getEvent(description)
+
+	if(!map) {
+		if(isZoneMessage(description)) {
+			map = parseIasMessage(description)
+		} else {
+			map = parseAttrMessage(description)
+		}
+	} 
+
+	def result = map ? createEvent(map) : [:]
+
+	if (description?.startsWith('enroll request')) {
+		def cmds = zigbee.enrollResponse()
+		log.debug "enroll response: ${cmds}"
+		result = cmds?.collect { new physicalgraph.device.HubAction(it)}
+	}
+
+	return result
+}
+
+private boolean isZoneMessage(description) {
+	return (description?.startsWith('zone status') || description?.startsWith('zone report'))
+}
+
+private Map parseIasMessage(String description) {
+	ZoneStatus zs = zigbee.parseZoneStatus(description)
+
+	def result = [:]
+    def value = ""
+	if(zs.isAlarm1Set() || zs.isAlarm2Set()) {    	
+		value = "siren"
+	} else {
+    	value = "off"
+	}
+	def text = "Alarm state: ${value}"
+	result = [name: "alarm", value: value, descriptionText: text, displayed: true]
+
+	return result
+}
+
+private Map parseAttrMessage(description) {
+	def descMap = zigbee.parseDescriptionAsMap(description)
+
+	def map = [:]
+	if(descMap.clusterInt == POLL_CONTROL_CLUSTER && descMap.commandInt == CHECK_IN_INTERVAL_CMD) {
+		sendCheckIntervalEvent()
+	} else if (descMap?.clusterInt == 0x0502 && descMap.attrInt == 0x0000) {
+    	state.warningDuration = descMap.value
+
+		map = [name: "warningDuration", value: state.warningDuration, descriptionText: "Maxwarning duration: ${state.warningDuration}", displayed: true]
+	} 
+
+	return map
+}
+
+private sendCheckIntervalEvent() {
+	sendEvent(name: "checkInterval", value: 30 * 60, displayed: false, data: [protocol: "zigbee", hubHardwareId: device.hub.hardwareID])
+}
+
+def ping() {
+	refresh()
+}
+
+def refresh() {
+	return zigbee.readAttribute(0x0502, 0x0000)
```

Comment:
you should probably ask for 

`zigbee.readAttribute(zigbee.IAS_ZONE_CLUSTER, zigbee.ATTRIBUTE_IAS_ZONE_STATUS)`

instead of the 0x0502 cluster, since that doesn't return anything incredibly relevant